gcp/iam/workload_identity_federation/trusted_issuers.yaml

```diff
@@ -0,0 +1,52 @@
+azure:
+  issuer: "https://sts.windows.net/${owner}"
+  attributes:
+    "google.subject": "assertion.sub"
+    "google.groups": "assertion.groups"
+bitbucket-pipelines:
+  issuer: "https://api.bitbucket.org/2.0/workspaces/${owner}/pipelines-config/identity/oidc"
+  allowed_audiences:
+    - "ari:cloud:bitbucket::workspace/${workspace_uuid}"
+  attributes:
+    "google.subject": "assertion.sub"
+    "attribute.workspace_uuid": "assertion.workspaceUuid"
+    "attribute.repository": "assertion.repositoryUuid"
+    "attribute.git_ref": "assertion.branchName"
+  condition: "assertion.workspaceUuid=='${workspace_uuid}'"
```

Comment:
`bitbucket-pipelines` has both `owner` and `workspace_uuid` because `owner` takes the form of the workspace name which is used in the issuer, but workspace uuid is used for the audience